wordpress-mobile/WordPress-Aztec-Android
# Issue/145 fixing activity state restoration

Fixes #145 

To test:
- Toggle "Don't keep activities" from developer options.
- Open Aztec demo.
- Press home button.
- Go back to Aztec demo from the recent apps list.
- Make sure that app does not crash and text in editor looks same and has cursor at same position.
